src/views/TruckSearch.vue

```diff
@@ -132,7 +132,7 @@ export default {
 	created() {
 		api.vehiclesApi.getVehicle('CYMS18010001')
 		.then(vehicle => {
-			vehicle.imageURL = 'https://lh3.googleusercontent.com/proxy/hr_WseNeJmg14dsuam2GwPHTZ90_0CM4GZJZ_q6rFbZsnVBKO5QK35BWFKSfSEVZqhRCqI4MoR7SyerJjtUxJNnWZTYmjSBjasbwBt1Ych9lcekYjjoIfm7ykTdIKC6IW54ovE-XOXzaZo2-B0QhpRKnjMCxdsDfyYAI';
+			vehicle.imageURL = 'https://www.cn-sinotruk.com/wp-content/uploads/2018/12/Sinotruk-STR-M5G-6x4-tractor-truck.jpg';
 			vehicle.motorConfig = '6x4';
 			this.allTrucks.push(vehicle);
```

Comment:
I can see that you are using the same URL in many places for debugging/demo purposes.
It would be a good practice to declare a constant (or even a common variable) at the beginning of the file containing the URL, and then invoking the variable when needed.
By doing so, future changes to the file will be done quicker and cleaner.

Reply:
Changes were applied